### DISCUSSION TITLE
Account Blocked

### DISCUSSION THREAD

**Discussion by Saobby** [vercel/community#1324] [Help]
### Summary

My account was automatically blocked because someone kept sending so many requests to my serverless functions deployed on Vercel that reached the limit of Vercel.What should I do to unblock?

![](https://e.saobby.com/uploads/2023/01/CD45654B_44FC_4919_90CE_00BAD66BD5A4.jpeg)

**Comment by Gtd232** [comment]
你好！我看你是中国人！我就用中文回答你！  
你好！我的账号也因为过多请求被锁，它提示我升级到Pro计划。  
![076e3fe319b51fc6f571f69c3673e09](https://user-images.githubusercontent.com/79702405/213909407-0b6520f2-3f2b-4675-ae1c-464d21880dfb.jpg)  
而且现在你的站，它提示：  
![image](https://user-images.githubusercontent.com/79702405/213909438-cb4bc2e1-d750-4cec-93b4-ae86184783e5.png)  
所以你要9999付费！  
我目前只知道这种方法。  
你可以套Cloudflare到你所有站点，尽管这样会使部分用户减速，但可以防御这些恶意请求，再把你的vercel.app的子域设置得复杂点。

**Reply by jexjws** [reply]
![20 USD](https://user-images.githubusercontent.com/65129712/213912455-fbc1ac8c-17cd-45c0-82a4-9dc473e2dfee.png)

135软妹币！看起来好桂 唔唔唔

This price is a bit expensive for individual developers who use it for non-commercial purposes.

**Reply by Saobby** [reply]
> 你好！我看你是中国人！我就用中文回答你！
> 你好！我的账号也因为过多请求被锁，它提示我升级到Pro计划。
> ![076e3fe319b51fc6f571f69c3673e09](https://user-images.githubusercontent.com/79702405/213909407-0b6520f2-3f2b-4675-ae1c-464d21880dfb.jpg)
> 而且现在你的站，它提示：
> ![image](https://user-images.githubusercontent.com/79702405/213909438-cb4bc2e1-d750-4cec-93b4-ae86184783e5.png)
> 所以你要9999付费！
> 我目前只知道这种方法。
> 你可以套Cloudflare到你所有站点，尽管这样会使部分用户减速，但可以防御这些恶意请求，再把你的vercel.app的子域设置得复杂点。

你好！太贵了！我买不起！

另外！套cloudflare只防君子不防小人！别人向`any.vercel.app`发送请求，并把请求头的`Host`字段改成你的域名就行了！这样就能不通过cloudflare直接访问你的网站，所以cloudflare没有用！他并不能给vercel提供防御！

我想到了一个方案，就是用cloudflare worker反向代理部署在vercel的程序，这样就不会泄漏vercel绑定的域名。

Translation:  
Hi! too expensive! I can not afford it!

in addition! Using cloudflare won't change anything.Attackers send a request to `any.vercel.app`, and set the `Host` field of the request header to your domain name, then they can directly access your website without going through cloudflare, so using cloudflare is useless!

I thought of a solution, which is to use cloudflare worker to reverse proxy the program deployed in vercel, so that the domain name bound to vercel will not be leaked.

**Reply by Gtd232** [reply]
> 
> 你好！太贵了！我买不起！
> 
> 另外！套cloudflare只防君子不防小人！别人向`any.vercel.app`发送请求，并把请求头的`Host`字段改成你的域名就行了！这样就能不通过cloudflare直接访问你的网站，所以cloudflare没有用！他并不能给vercel提供防御！
> 
> 我想到了一个方案，就是用cloudflare worker反向代理部署在vercel的程序，这样就不会泄漏vercel绑定的域名。

你好！我也觉得很贵！所以我重新注册了一个账号！只不过要消耗一个手机号码！  
我觉得你的方案9999好！
另外，我现在买了一个派！它可以当小型服务器！加上内网穿透，别人可以9999访问！

Translated with DeepL.
Hello! I thought it was expensive too! So I re-registered an account! Only to consume a mobile number!  
I think your plan is good!
BTW, I bought a Pi now! It can be a mini server! Plus intranet penetration so others can access!

**Reply by Gtd232** [reply]
文档中说锁了可以上诉（https://vercel.com/guides/why-is-my-account-deployment-blocked）  
另外，当时我的站点接到大量请求时有发邮件过来提醒我的
